apps/example-server/src/main/documents/documents.routes.ts

```diff
@@ -30,3 +39,6 @@ export const documentV1Route = createRouter()
     .openapi(deleteDocumentRoute, deleteDocumentHandler)
     .openapi(deleteAllDocumentRoute, deleteAllDocumentHandler)
     .openapi(replaceFileRoute, replaceFileHandler)
+    .openapi(getFilesByEntityNameRoute, getFilesByEntityNameHandler)
+    .openapi(getFilesByEntityNameAndIdRoute, getFilesByEntityNameAndIdHandler)
+    .openapi(getFilesByGroupIdRoute, getFilesByEntityNameAndIdHandler)
```

Comment:
All three of these handlers should be merged into one. In the handler we should check which queries are provided and take action based on that. Any query can be optional.